lib/routes/account.js

```diff
@@ -959,10 +980,13 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+          return db.sessionTokenVerified(sessionToken.id)
+          .then(function(token) {
+            return {
+              email: token.email,
+              verified: !!(token.emailVerified && token.tokenVerified)
```

Comment:
I understand why we're doing it, but it makes me sad what we're doing to the API here, telling a little white lie about the verification status of the account :-(

I don't have a better suggestion though.